src/js/hca/config/form.js

```diff
@@ -729,12 +730,16 @@ const formConfig = {
             isMedicaidEligible: {
               'ui:title': 'Are you eligible for Medicaid?',
               'ui:widget': 'yesNo',
-              'ui:help': 'Medicaid is a United States health program for eligible individuals and families with low income and few resources.'
+              'ui:description': <AdditionalInfo triggerText="Learn more about Medicaid.">
```

Comment:
I would be interested to see what this looks like when there's an error with the field.

Reply:
Great point @jbalboni, screenshot provided.